assemblies/assembly-getting-started-kogito-microservices.adoc

```diff
@@ -33,6 +33,20 @@ include::{enterprise-dir}/getting-started-kogito/proc-kogito-microservice-runnin
 
 include::{enterprise-dir}/getting-started-kogito/proc-kogito-microservice-interacting-app.adoc[leveloffset=+1]
 
+include::{enterprise-dir}/getting-started-kogito/proc-kogito-microservices-deploying-on-openshift.adoc[leveloffset=+1]
+include::{enterprise-dir}/getting-started-kogito/con-kogito-microservices-on-ocp.adoc[leveloffset=+2]
+include::{enterprise-dir}/getting-started-kogito/con-kogito-probes.adoc[leveloffset=+2]
```

Comment:
I would consider information about probes as kind of additional or side information. IMHO it should be placed in the latter part of docs, maybe above troubleshooting section?

Reply:
Done